### PR TITLE
Fix github url strings (org edx -> openedx)

### PR DESCRIPTION
**Pull request by sarina**
## This PR was autogenerated

This pr replaces the old GitHub organization, github.com/edx, with the new GitHub organization, github.com/openedx.

Tagging @openedx/tcril-engineering for review, but others are welcome to provide review.

Ref: https://github.com/openedx/tcril-engineering/issues/42

### CODE REVIEW

**Comment by sarina** [comment]
hi @pomegranited sorry to tag you 😛 

this diff is crazy but note that a huge portion is just the preamble on files:

image

Thank you & good luck!

**Comment by pomegranited** [comment]
@sarina heh thanks 😆 I'll review this next week.

File: util/create_data_czar/assign_czar_org_groups.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 assign_czar_org_groups.py
 
 Assigns data czars to the iam groups for their org based on the configuration specificed in the analytics-exporter
-repository, https://github.com/edx/analytics-exporter/blob/master/config/default.yaml.
 
 The file needs to be available locally as the repo is private.
```
Comment (pomegranited) [inline util/create_data_czar/assign_czar_org_groups.py]: The file to reference here has moved, so this line need not be deleted:

> repository, https://github.com/openedx/edx-analytics-exporter/blob/master/sample-config.yaml

And the repo is no longer private, so this line can be removed:

> The file needs to be available locally as the repo is private.

**Review by pomegranited — APPROVED** [review]
:+1: -- one link that can be changed instead of deleted, but this is good to go once that's addressed.

So glad you scripted all of these @sarina :)

- [x] I tested this by reading through the updated files and checking that all the links work.
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] Updates documentation
- [x] Commit structure follows [OEP-0051](https://github.com/edx/open-edx-proposals/pull/189)

**Comment by sarina** [comment]
thanks @pomegranited 😄

**Pull request merged**